corehq/apps/cleanup/management/commands/remove_deleted_cases_from_es.py

```diff
@@ -0,0 +1,52 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.es import CaseES, CaseSearchES, case_adapter
+from corehq.form_processor.models.cases import CommCareCase, RebuildWithReason
+from corehq.form_processor.backends.sql.processor import FormProcessorSQL
+
+
+class Command(BaseCommand):
+    help = ('Remove cases from case index in elasticsearch if deleted')
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('--dry-run', action="store_true")
+
+    def handle(self, domain, **options):
+        dry_run = options["dry_run"]
+        missing_case_ids = get_case_ids_missing_from_case_search(domain)
+        for case_id in missing_case_ids:
+            should_delete = should_case_be_deleted(case_id, domain)
+            if should_delete:
+                print(f"removing case {case_id} in domain {domain} from ES")
+                if not dry_run:
+                    case_adapter.delete(case_id, refresh=True)
+            else:
+                print(f"case {case_id} in domain {domain} is not deleted")
```

Comment:
This isn't blocking but if the case isn't deleted shouldn't the case be migrated into the casesearch index? Or will that be for a future PR?

Reply:
Good question. It'll definitely be a future PR, but whether or not it is re-adding those cases to the casesearch index, or finding something else out about the remaining non-deleted cases that implies they should not be in the case index is still unknown. Those are the two possibilities at this point though.